### PR TITLE
Create execute-dotnet-assembly-via-clr-host.yml

### LINKED ISSUE #417: execute .NET assembly via CLR host
https://github.com/TheWover/donut/blob/master/DonutTest/rundotnet.cpp

### PR DESCRIPTION
**Pull request by bkojusner**
This rule passes the thorough linting test and closes #417

### CODE REVIEW

File: load-code/dotnet/execute-dotnet-assembly-via-clr-host.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - bytes: 22 67 2F CB 3A AB D2 11 9C 40 00 C0 4F A3 0A 3E = IID_ICorRuntimeHost
+      - api: CorBindToRuntime
+      - basic block:
+        - or:
+          - and:
+            - operand[0].offset: 0x50 = Start
+            - characteristic: indirect call
+          - and:
+            - operand[0].offset: 0x68 = GetDefaultDomain
+            - characteristic: indirect call
```
Comment (mr-tz) [inline load-code/dotnet/execute-dotnet-assembly-via-clr-host.yml:26]: Are both Start and GetDefaultDomain required? What about potential other offsets, like Load_3, get_EntryPoint, or Invoke_3?

**Review by mr-tz — APPROVED** [review]
this looks great, one question on the chosen offsets, but not a stopper for me

File: load-code/dotnet/execute-dotnet-assembly-via-clr-host.yml (hunk 1 above)
Comment (bkojusner) [reply load-code/dotnet/execute-dotnet-assembly-via-clr-host.yml]: Block is optional, removed it and left two byte sequences and two API calls. Thanks!

**Review by mike-hunhoff — APPROVED** [review]
LGTM

**Pull request merged**